checks.d/linux_proc_extras.py

```diff
@@ -0,0 +1,83 @@
+# project
+from checks import AgentCheck
+from utils.subprocess_output import get_subprocess_output
+import collections
+import subprocess
+
+PROCESS_STATES = {
+    'D': 'uninterruptible',
+    'R': 'runnable',
+    'S': 'sleeping',
+    'T': 'stopped',
+    'W': 'paging',
+    'X': 'dead',
+    'Z': 'zombie',
+}
+
+PROCESS_PRIOS = {
+    '<': 'high',
+    'N': 'low',
+    'L': 'locked'
+}
+
+class MoreUnixCheck(AgentCheck):
+    def check(self, instance):
+        tags = instance.get('tags', [])
+
+        state_counts = collections.Counter(
+            uninterruptible=0,
```

Comment:
The good thing about the Counter is that you don't need to initialize your keys at 0 :)
```python
state_counts = Counter()
```
would give you the same result